src/CompanyService.Business/Commands/Department/CreateDepartmentCommand.cs

```diff
@@ -45,22 +45,27 @@ public OperationResultResponse<Guid> Execute(CreateDepartmentRequest request)
 
             _validator.ValidateAndThrowCustom(request);
 
-            var company = _companyRepository.Get();
+            OperationResultResponse<Guid> response = new();
+
+            DbCompany company = _companyRepository.Get();
 
             if (company == null)
             {
-                return new OperationResultResponse<Guid>
-                {
-                    Status = OperationResultStatusType.Failed,
-                    Errors = new List<string> () { "Company does not exist" }
-                };
+                response.Status = OperationResultStatusType.Failed;
+                response.Errors.Add("Company does not exist, please create company");
```

Comment:
Does it work? Seems like Errors is not assigned and at this moment it is null

Reply:
yes)